fe/fe-core/src/main/java/org/apache/doris/qe/ConnectContext.java

```diff
@@ -557,6 +563,11 @@ public void checkTimeout(long now) {
 
                 // Only kill
                 killFlag = true;
+            } else if (delta > userQueryTimeout * 1000) {
```

Comment:
If querytimeout is set in property, it should override the querytimeou in session variable

Reply:
done